report/computer_last_inventory.php

```diff
@@ -37,7 +37,7 @@
 
 include("../../../inc/includes.php");
 
-Html::header(__('FusionInventory', 'glpiinventory'), $_SERVER['PHP_SELF'], "utils", "report");
+Html::header(__('GLPIInventory', 'glpiinventory'), $_SERVER['PHP_SELF'], "utils", "report");
```

Comment:
```suggestion
Html::header(__('GLPI Inventory', 'glpiinventory'), $_SERVER['PHP_SELF'], "utils", "report");
```

(string with a space already exists in translations)